src/pages/main/HomePage/components/HomeFooter.tsx

```diff
@@ -0,0 +1,150 @@
+import styled from 'styled-components';
+
+import { CarouselLeftIcon, CarouselrightIcon } from '../../../../assets/svgs';
+import { FooterCarouselItemInfo, FooterCaureselTitle } from '../constants';
+
+const HomeFooter = () => {
+  return (
+    <HomeFooterWrapper>
+      <FooterTitle>다음 여행을 위한 추천 여행지</FooterTitle>
+      <FooterCarouselTitleList>
+        {FooterCaureselTitle.map((info) => {
+          return <FooterCarouselTitleItem key={info.id}>{info.text}</FooterCarouselTitleItem>;
+        })}
+      </FooterCarouselTitleList>
+      <FooterCarousel>
+        <CarouselLeftBtn />
+        {FooterCarouselItemInfo.map((info) => {
+          return (
+            <FooterCarouselItem key={info.id}>
+              <FooterCarouselImg src={info.img} />
+              <FooterCarouselInfo>
+                <FooterCarouselTitle>{info.title}</FooterCarouselTitle>
+                <FooterCarouselRecommend>{info.recommend}</FooterCarouselRecommend>
+                <FooterCarouselDescription>{info.description}</FooterCarouselDescription>
+                <FooterCarouselPriceWrapper>
+                  <FooterCarouselDefaultPrice>{info.defaultPrice}</FooterCarouselDefaultPrice>
+                  <FooterCarouselDiscountPrice>{info.discountPrice}</FooterCarouselDiscountPrice>
+                  <FooterCarouselDay>{info.day}</FooterCarouselDay>
+                </FooterCarouselPriceWrapper>
+              </FooterCarouselInfo>
+            </FooterCarouselItem>
+          );
+        })}
```

Comment:
오호 여기 Return 안쓰고 사용할 수 있어요 !
```suggestion
        {FooterCarouselItemInfo.map((info) => (
            <FooterCarouselItem key={info.id}>
              <FooterCarouselImg src={info.img} />
              <FooterCarouselInfo>
                <FooterCarouselTitle>{info.title}</FooterCarouselTitle>
                <FooterCarouselRecommend>{info.recommend}</FooterCarouselRecommend>
                <FooterCarouselDescription>{info.description}</FooterCarouselDescription>
                <FooterCarouselPriceWrapper>
                  <FooterCarouselDefaultPrice>{info.defaultPrice}</FooterCarouselDefaultPrice>
                  <FooterCarouselDiscountPrice>{info.discountPrice}</FooterCarouselDiscountPrice>
                  <FooterCarouselDay>{info.day}</FooterCarouselDay>
                </FooterCarouselPriceWrapper>
              </FooterCarouselInfo>
            </FooterCarouselItem>
          ))}
```

Reply:
앗 그렇네요???? 저는 한줄 이상 넘어가면 return을 써야된다고만 생각해서 return 써줬는데 이친구는 그냥 한줄의 코드로 인식되니까 return 안써줘도 되겠네요 배워갑니다!!!!